torch/package/importer.py

```diff
@@ -144,7 +153,30 @@ def load_pickle(self, package: str, resource: str, map_location=None) -> Any:
             Any: the unpickled object.
         """
         pickle_file = self._zipfile_path(package, resource)
-        return _load(self.zip_reader, map_location, self, pickle_file=pickle_file)
+        restore_location = _get_restore_location(map_location)
+
+        def persistent_load(saved_id):
+            assert isinstance(saved_id, tuple)
+            typename = _maybe_decode_ascii(saved_id[0])
+            data = saved_id[1:]
+
+            assert typename == 'storage', \
+                f"Unknown typename for persistent_load, expected 'storage' but got '{typename}'"
+            data_type, key, location, size = data
+            if key not in self._loaded_storages:
```

Comment:
if you call `load_pickle` twice on the same resource you will get different copies of the model, but they will share tensor data. That seems inconsistent because it is neither 100% a copy nor 100% an alias. My intuition is that the tensors should be copies across different calls to load.